### PR TITLE
upgrade terraform v1.0.4

### PR DESCRIPTION
**Pull request by marcoldp**

### CODE REVIEW

File: terraform/versions.tf [hunk 1]
```diff
@@ -1,4 +1,4 @@
 
 terraform {
-  required_version = ">= 0.12"
+  required_version = ">= 1.0.3"
```
Comment (sdurrheimer) [inline terraform/versions.tf:3]: I would put `1.0` here

Comment (marcoldp) [reply terraform/versions.tf:3]: It was to have the last available version but i can change

Comment (sdurrheimer) [reply terraform/versions.tf:3]: The restriction of the patch version `.3` makes no sense, it's only bug fixes.
What is interesting is to say: you need a version of 1.0 at least.

Comment (marcoldp) [reply terraform/versions.tf:3]: ok i'll change :)

**Pull request closed without merging**